config/config.exs

```diff
@@ -5,4 +5,4 @@ use Mix.Config
       {"localhost", 9093},
       {"localhost", 9094},
     ],
-    consumer_group: false
+    consumer_group: "kafka_ex_dev"
```

Comment:
Those code paths _should not_ (consumer groups did not fully exist prior to kafka 0.8.2) be called by users of kafka < 0.8.2, I assumed this was documented but I probably missed it.